tests/TestCase/Model/Behavior/UploadBehaviorTest.php

```diff
@@ -71,7 +71,7 @@ public function testInitialize()
             ->onlyMethods(['setColumnType'])
             ->disableOriginalConstructor()
             ->getMock();
-        $schema->expects($this->once())
+        $schema->expects($this->any())
```

Comment:
Instead of changing this add the `field` column to the schema.

Reply:
Do you mean adding the `field` column in `tests/schema.php`? Doing this does nothing by itself, and test are failing:

> 1) Josegonzalez\Upload\Test\TestCase\Model\Behavior\UploadBehaviorTest::testInitialize
> Expectation failed for method name is "setColumnType" when invoked 1 time.
> Method was expected to be called 1 time, actually called 0 times.

What I understand is that, as the schema is a mock, I have to mock the 'hasColumn' method as well, and return true.

```
$schema = $this->getMockBuilder('Cake\Database\Schema\TableSchema')
   ->onlyMethods(['setColumnType', 'hasColumn'])
   ->disableOriginalConstructor()
   ->getMock();
$schema->expects($this->once())
   ->method('hasColumn')
   ->with('field')
   ->willReturn(true);
```

The test then passes, but then I don't see the point of modifying the schema in that case, or is there something else that I'm missing?